business/checkers/destinationrules/multi_match_checker.go

```diff
@@ -70,6 +75,28 @@ func (m MultiMatchChecker) Check() models.IstioValidations {
 	return validations
 }
 
+func enablesNonLocalmTLS(dr kubernetes.IstioObject, fdqn Host) bool {
```

Comment:
This is nitpicking only, but considering there's no use for the full ``Host``, passing it here was perhaps a bit unnecessary.